crypto/impl_dispatch_test.cc

```diff
@@ -34,10 +34,9 @@ class ImplDispatchTest : public ::testing::Test {
  public:
   void SetUp() override {
 #if defined(OPENSSL_X86) || defined(OPENSSL_X86_64)
-    aesni_ = OPENSSL_ia32cap_P[1] & (1 << (57 - 32));
-    avx_movbe_ = ((OPENSSL_ia32cap_P[1] >> 22) & 0x41) == 0x41;
-    ssse3_ = OPENSSL_ia32cap_P[1] & (1 << (41 - 32));
-    sha_ext_ = OPENSSL_ia32cap_P[2] & (1 << 29);
```

Comment:
function for shaext capability should be added (see the comment in sha512-x86_64.pl